library/src/test/resources/pyPfbOutput/showSchema/minimal_schema.json

```diff
@@ -1,296 +1 @@
-[
```

Comment:
This was the only file that was pretty formatted, but I thought it'd make more sense to have what's checked in match what's generated by the script.

Can add a pretty output format for all if there's an appetite for that.

Reply:
I feel like its so easy to get the pretty formatted version, I think it makes sense to have whats being generated so there is not confusion.